.circleci/config.yml

```diff
@@ -81,6 +81,17 @@ jobs:
                               --out test-results/rspec/rspec.xml \
                               --format progress \
                               -- ${TESTFILES}
+      - run:
+          name: "eslint"
+          command: "yarn lint"
+      - run:
+          name: "Jest: Install junit coverage" 
```

Comment:
Questions about this and CI: 

- Does this integrate with Code Climate test coverage?
- Does a failure in `Jest: Tests` show up/alert CircleCI?

Reply:
Ya I followed the suggestions here to get the junit formatter setup for circle: https://circleci.com/docs/2.0/collect-test-data/#jest

Ya any Jest spec failure will prevent the build from passing